cvat/apps/engine/permissions.py

```diff
@@ -750,53 +753,53 @@ def get_scopes(request, view, obj):
     def get_resource(self):
         data = None
         if self.obj:
+            organization_id = None
```

Comment:
This assignment is unnecessary, since both branches of the `if` below assign a new value.